src/platform/datapath_raw_socket.c

```diff
@@ -872,27 +1317,50 @@ CxPlatFramingWriteHeaders(
     CXPLAT_DBG_ASSERT(
         Family == QUIC_ADDRESS_FAMILY_INET || Family == QUIC_ADDRESS_FAMILY_INET6);
 
-    //
-    // Fill UDP header.
-    //
-    UDP->DestinationPort = Route->RemoteAddress.Ipv4.sin_port;
-    UDP->SourcePort = Route->LocalAddress.Ipv4.sin_port;
-    UDP->Length = QuicNetByteSwapShort((uint16_t)Buffer->Length + sizeof(UDP_HEADER));
-    UDP->Checksum = 0;
+    if (Socket->UseTcp) {
+        //
+        // Fill TCP header.
+        //
+        TCP = (TCP_HEADER*)(Buffer->Buffer - sizeof(TCP_HEADER));
+        TCP->DestinationPort = Route->RemoteAddress.Ipv4.sin_port;
+        TCP->SourcePort = Route->LocalAddress.Ipv4.sin_port;
+        TCP->Window = QuicNetByteSwapShort(65535);
```

Comment:
Can we just hard code the hex value instead of always calling the byteswap (which is still a func call I think).

Reply:
Yeah, should just write 0xFFFF. Swapping makes no difference.